server/channels/api4/group.go

```diff
@@ -301,6 +302,14 @@ func patchGroup(c *Context, w http.ResponseWriter, r *http.Request) {
 	auditRec.AddEventResultState(group)
 	auditRec.AddEventObjectType("group")
 
+	c.App.Srv().GetTelemetryService().SendTelemetryForFeature(
+		telemetry.TrackGroupsFeature,
+		"modify_group_edit_details",
+		map[string]any{
+			"user_actual_id": c.AppContext.Session().UserId,
```

Comment:
Perhaps create constants for common property names in order to reduce probability of typo errors.

Reply:
Thinking out loud. Since `user_actual_id` is pretty common in most events, is it possible to automatically add it to the properties?